tools/oam-converter/pkg/app/confdata.go

```
@@ -5,201 +5,280 @@ package app
-	coherence "github.com/verrazzano/verrazzano/tools/oam-converter/pkg/resources/coherenceresources"
-	helidon "github.com/verrazzano/verrazzano/tools/oam-converter/pkg/resources/helidonresources"
-	weblogic "github.com/verrazzano/verrazzano/tools/oam-converter/pkg/resources/weblogicresources"
-	extract "github.com/verrazzano/verrazzano/tools/oam-converter/pkg/traits"
+	consts "github.com/verrazzano/verrazzano/tools/oam-converter/pkg/constants"
+	"github.com/verrazzano/verrazzano/tools/oam-converter/pkg/resources"
+	"github.com/verrazzano/verrazzano/tools/oam-converter/pkg/traits"
+	"github.com/verrazzano/verrazzano/tools/oam-converter/pkg/types"
 	"io/ioutil"
-	"log"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"os"
+	"path/filepath"
 	"sigs.k8s.io/yaml"
 	"strings"
 )
 
-// contains function checks if the specifies string is present in array of strings
-func contains(arr []string, target string) bool {
-	for _, element := range arr {
-		if element == target {
-			return true
-		}
-	}
-	return false
-}
-
 func ConfData() error {
 
-	helidonWorkloads := []*vzapi.VerrazzanoHelidonWorkload{}
-	coherenceWorkloads := []*vzapi.VerrazzanoCoherenceWorkload{}
-	//Read app File
-	appData, err := ioutil.ReadFile("../../examples/bobs-books/bobs-books-app.yaml")
-	if err != nil {
-		fmt.Println("Failed to read YAML file:", err)
-		return err
-	}
-
-	//Read Comp file
-	compData, err := ioutil.ReadFile("../../examples/bobs-books/bobs-books-comp.yaml")
-	if err != nil {
-		fmt.Println("Failed to read YAML file:", err)
-		return err
-	}
-	//A map for app file
-	appMap := make(map[string]interface{})
+	var inputDirectory string
+	var outputDirectory string
 
-	// Unmarshal the OAM YAML input data into the map
-	err = yaml.Unmarshal(appData, &appMap)
-	if err != nil {
-		log.Fatalf("Failed to unmarshal YAML: %v", err)
+	//Check the length of args
+	if len(os.Args) != 3 {
+		fmt.Println("Not enough args to run tool")
```

we should return an error here